lib/cmd/fh3/resources/list.js

```diff
@@ -0,0 +1,21 @@
+/* globals i18n */
+var fhc = require("../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('List Resources across RHMAP'),
```

Comment:
I'd like for now keep a similar label that it had before and just change the Feedhenrry for RHMAP. Let's wait for the rest service be fixed and we have more information about the result to improve it. What do you think? Could we agree to move on in this way?